### PR TITLE
Nelsonomotuyole.json

### PR DESCRIPTION
**Pull request by Nelsonomotuyole**

### CODE REVIEW

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: Nelsonomotuyole.json
Comment (Aadarsh805) [inline Nelsonomotuyole.json:7]: `Links` should be `links`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: Nelsonomotuyole.json
Comment (Aadarsh805) [inline Nelsonomotuyole.json:16]: this GitHub icon should be `FaGithub`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: Nelsonomotuyole.json
Comment (Nelsonomotuyole) [reply Nelsonomotuyole.json:6]: how do I correct it sir

File: Nelsonomotuyole.json
Comment (Nelsonomotuyole) [reply Nelsonomotuyole.json:7]: yes how do i coorect the mistakes

Comment (Aadarsh805) [reply Nelsonomotuyole.json:7]: > yes how do i coorect the mistakes

You can make another commit with the changes requested on the same branch you did earlier
Did you did it in GitHub or using a code editor?

Comment (Nelsonomotuyole) [reply Nelsonomotuyole.json:7]: i did it with GitHub

Comment (Aadarsh805) [reply Nelsonomotuyole.json:7]: > i did it with GitHub

so you do the exact same thing again with these new changes and commit again

Comment (Aadarsh805) [reply Nelsonomotuyole.json:7]: go to your forked repo, find your file, edit the changes and commit

Comment (Nelsonomotuyole) [reply Nelsonomotuyole.json:7]: When I make changes should I commit directly to the to patch-1 branch or creat a new branch

Comment (Aadarsh805) [reply Nelsonomotuyole.json:7]: > When I make changes should I commit directly to the to patch-1 branch or creat a new branch

make it on the same `patch-` branch

Comment (Nelsonomotuyole) [reply Nelsonomotuyole.json:7]: Okay thanks you so much really appreciate

File: Nelsonomotuyole.json
Comment (Aadarsh805) [reply Nelsonomotuyole.json:16]: > Sorry Is it 'faGitHub'

nah its `FaGithub` with a small `h`

Comment (Nelsonomotuyole) [reply Nelsonomotuyole.json:16]: Okay thanks
When it has been merged and approved where do I find the url for my website sir?

Comment (Aadarsh805) [reply Nelsonomotuyole.json:16]: > Okay thanks When it has been merged and approved where do I find the url for my website sir?

after merge and the release your URL will be `https://linkfree.eddiehub.io/Nelsonomotuyole`

**Comment by Aadarsh805** [comment]
@Nelsonomotuyole one more thing, your file must be in the `data` directory, for this just change the name for the file to `data/
Nelsonomotuyole.json`

File: Nelsonomotuyole.json
Comment (Aadarsh805) [inline Nelsonomotuyole.json]: one more typo I found here, it should be `python` instead of `phyton`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: Nelsonomotuyole.json
Comment (Nelsonomotuyole) [reply Nelsonomotuyole.json]: okay thanks sir

File: Nelsonomotuyole.json
Comment (Nelsonomotuyole) [reply Nelsonomotuyole.json]: its telling me not found on the browser without showing my credentials

File: data/Nelsonomotuyole.json
Comment (Aadarsh805) [inline data/Nelsonomotuyole.json:5]: still not `python`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: Nelsonomotuyole.json
Comment (Aadarsh805) [reply Nelsonomotuyole.json]: > i posted the link on a browser but it did not show anything i think it hasnt been approved yet?

yes you need to wait for this to get PR to get merged and released in a new version of Linkfree, releases are pushed every day, so just wait for the merge

File: data/Nelsonomotuyole.json
Comment (Nelsonomotuyole) [reply data/Nelsonomotuyole.json]: corrected thanks

File: Nelsonomotuyole.json
Comment (Nelsonomotuyole) [reply Nelsonomotuyole.json]: okay thanks

File: data/Nelsonomotuyole.json
Comment (Panquesito7) [inline data/Nelsonomotuyole.json:25]: These extra lines should be removed.
```suggestion
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data/Nelsonomotuyole.json
Comment (Nelsonomotuyole) [reply data/Nelsonomotuyole.json:25]: Thanks very much,will do that 👍

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

Alos, Shoutout to @Aadarsh805 for explaining this so well and helping out

<p align="center"></p>

**Comment by Nelsonomotuyole** [comment]
Please how can I edit this PR?

**Comment by Aadarsh805** [comment]
> Please how can I edit this PR?

If you wanna edit some changes to your profile or update it, you should make a brand new pr for that saying `updated:<your username>.json`
Make sure to have your repo synced with the original repo before making changes
After syncing with the original you'll find your file in the same `data` directory
Make changes in there and push to a new branch

**Comment by Aadarsh805** [comment]
@Nelsonomotuyole if you get stuck or need some help, feel free to ask 😇

**Comment by Nelsonomotuyole** [comment]
Alright thanks so much

**Pull request merged**
